src/user-icon/user-icon.scss

```diff
@@ -4,57 +4,47 @@
 @import '../style/index';
 @import '../text/mixins';
 
-@mixin iui-user-icons($size: $iui-icons-xl) {
+@mixin iui-user-icon {
   @include iui-reset;
-
   @include iui-font-family;
   display: inline-block;
   position: relative;
-  width: $size;
-  height: $size;
   margin: 0;
   user-select: none;
 
-  > .iui-user-icons-stroke {
+  > .iui-stroke {
     display: block;
     position: absolute;
     z-index: 2;
     left: 0;
     top: 0;
-    width: $size;
-    height: $size;
-    line-height: $size;
+    width: 100%;
+    height: 100%;
     margin: 0;
     border-radius: 50%;
     box-sizing: border-box;
-
     @include themed {
       box-shadow: inset 0 0 0 2px
         rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
     }
   }
 
-  > .iui-user-icons-initials {
-    @include iui-text(leading);
-
+  > .iui-initials {
     display: block;
     position: absolute;
-
     z-index: 0;
     left: 0;
     top: 0;
-    width: $size;
-    height: $size;
-    line-height: $size;
+    width: 100%;
+    height: 100%;
     margin: 0;
     border-radius: 50%;
-
-    text-align: center;
+    font-style: normal;
```

Comment:
What's this for?

Reply:
In the table demo the user icon was inheriting the italic styling from a disabled table row.  With `font-style: normal;` the user icon no longer becomes italic.